cdap-docs/reference-manual/source/http-restful-api/logging.rst

```diff
@@ -168,11 +168,11 @@ Changing Program Log Levels
 Log levels can be set for a particular run of a program. Once changed, they can be reset back to what
 they started with by using the :ref:`reset endpoint <http-restful-api-logging-resetting>` shown below.
 
-**Note:** The log levels can only be changed for programs that are running under CDAP Distributed mode.
+**Note:** The log levels can only be changed for programs that are running under Distributed CDAP mode.
```

Comment:
I think ``mode`` does not make sense if you switch the order of Distributed and CDAP